RickyChien.md

```diff
@@ -1,8 +1,11 @@
-##  ~ 07/15
+##  ~ 07/22
 [Firefox]
```

Comment:
Could you split specify which part of Firefox into sections?

Like the one below is control center and the one after that is DevTools. Thanks!